### DISCUSSION TITLE
Generating server classes for Quarkus

### DISCUSSION THREAD

**Discussion by ghost** [quarkiverse/quarkus-openapi-generator#155] [Ideas]
We are using contract first for API design in our Spring Boot project quite heavily. We are considering to use Quarkus however there is not a single working openapi generator for Quarkus. Do you think it would be a bad idea to support this besides generation of API classes for client? Eventually, can I do it with this project already now? I can help if needed.

**Comment by hbelmiro** [comment]
Sounds interesting...
Have you thought about how the user would choose between generating server or API code when using this extension?

@ricardozanini WDYT?

**Comment by ricardozanini** [comment]
@martinformi this is possible since the underlying generator already supports it. I'd say to open an issue with a few implementation ideas/details of how we can achieve that.

**Comment by ricardozanini** [comment]
Already supported in the new package `server`.
